osu.Game.Rulesets.Osu/Difficulty/Evaluators/AimEvaluator.cs

```diff
@@ -36,22 +37,23 @@ public static double EvaluateDifficultyOf(DifficultyHitObject current, bool with
             var osuLastLastObj = (OsuDifficultyHitObject)current.Previous(1);
             var osuLast2Obj = (OsuDifficultyHitObject)current.Previous(2);
 
-            const int radius = OsuDifficultyHitObject.NORMALISED_RADIUS;
             const int diameter = OsuDifficultyHitObject.NORMALISED_DIAMETER;
 
             // Calculate the velocity to the current hitobject, which starts with a base distance / time assuming the last object is a hitcircle.
-            double currVelocity = osuCurrObj.LazyJumpDistance / osuCurrObj.AdjustedDeltaTime;
+            double currDistance = osuCurrObj.LazyJumpDistance + calculateSnappingDifficulty(osuCurrObj.LazyJumpDistance, osuCurrObj, osuLastObj);
```

Comment:
This is arguably semantics but for clarity and sanity's sake, could we change this so that rather than adding onto `currDistance` etc., we have a base difficulty variable that is equal to the inherent snapping distance and then add everything else on instead? To me it would be way clearer that the intent is to deal with snapping having an inherent difficulty